### ISSUE TITLE
Correct device name field to not show the email of the user, but to rather show device name

### ISSUE THREAD

**Issue by danielgospodinow** [qbaware/homeassistant-eldom#15]
<img width="1053" alt="image" src="https://github.com/user-attachments/assets/dab6a17e-7ff7-4979-bb43-49cc3d66796e">

**Comment by danielgospodinow** [comment]
Here's an example of how it should be:
<img width="1382" alt="image" src="https://github.com/user-attachments/assets/772e56d2-7707-4348-b48f-446f9b9485fa">

<img width="1382" alt="image" src="https://github.com/user-attachments/assets/4e3e084c-77ff-4b5c-adbc-c12b140e7997">

**Comment by danielgospodinow** [comment]
Seems to be an easy fix: 

```python
        return DeviceInfo(
            name=self._water_heater.name,
        )
```

The `name` prop of the `DeviceInfo` class should be populated.

**Comment by danielgospodinow** [comment]
Fixed with [1.0.0](https://github.com/qbaware/homeassistant-eldom/releases/tag/1.0.0).
